mate-window-picker-applet/task-title.c

```diff
@@ -454,27 +572,17 @@ task_title_init (TaskTitle *title)
   g_signal_connect (priv->button, "draw",
                     G_CALLBACK (on_button_draw), title);
 
+  // Initialize button_image to allow easy setting by icon name after that.
   gdkscreen = gtk_widget_get_screen (GTK_WIDGET (title));
   theme = gtk_icon_theme_get_for_screen (gdkscreen);
-  gtk_icon_size_lookup (GTK_ICON_SIZE_MENU,
-                        &width, &height);
-
-  priv->quit_icon = gtk_icon_theme_load_icon (theme, "mate-logout", width, 0, NULL);
-
+  gtk_icon_size_lookup (GTK_ICON_SIZE_MENU, &width, &height);
+  priv->quit_icon = gtk_icon_theme_load_icon (theme, "gnome-logout", width, 0, NULL);
```

Comment:
Suggestion: use `system-log-out` icon name, as in mate-session-manager.

Reply:
Yes good idea. I didn't find this icon at first sight. I just saw that "mate-logout" didn't really exists, that's why I replace it with gnome-logout. I've made the change and push again.